packages/site/src/components/Buttons.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentProps } from 'react';
+import styled from 'styled-components';
+import { MetamaskState } from '../hooks';
+import { ReactComponent as FlaskFox } from '../assets/flask_fox.svg';
+
+const Link = styled.a`
+  display: flex;
+  align-self: flex-start;
+  align-items: center;
+  justify-content: center;
+  font-size: ${(props) => props.theme.fontSizes.small};
+  border-radius: ${(props) => props.theme.radii.button};
+  border: 1px solid ${(props) => props.theme.colors.background.inverse};
+  background-color: ${(props) => props.theme.colors.background.inverse};
+  color: ${(props) => props.theme.colors.text.inverse};
+  text-decoration: none;
+  font-weight: bold;
+  padding: 10px;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+
+  &:hover {
+    background-color: transparent;
+    border: 1px solid ${(props) => props.theme.colors.background.inverse};
+    color: ${(props) => props.theme.colors.text.default};
+  }
+
+  ${({ theme }) => theme.mediaQueries.small} {
+    width: 100%;
+    box-sizing: border-box;
+  }
+`;
+
+const Button = styled.button`
+  display: flex;
+  align-self: flex-start;
+  align-items: center;
+  justify-content: center;
+  ${({ theme }) => theme.mediaQueries.small} {
+    width: 100%;
+  }
+`;
+
+const ButtonText = styled.span`
+  margin-left: 10px;
+`;
+
+const ConnectedContainer = styled.div`
+  display: flex;
+  align-self: flex-start;
+  align-items: center;
+  justify-content: center;
+  font-size: ${(props) => props.theme.fontSizes.small};
+  border-radius: ${(props) => props.theme.radii.button};
+  border: 1px solid ${(props) => props.theme.colors.background.inverse};
+  background-color: ${(props) => props.theme.colors.background.inverse};
+  color: ${(props) => props.theme.colors.text.inverse};
+  font-weight: bold;
+  padding: 12px;
+`;
+
+const ConnectedIndicator = styled.div`
+  content: ' ';
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  background-color: green;
+`;
+
+export const InstallFlaskButton = () => (
+  <Link href="https://metamask.io/flask/" target="_blank">
+    <FlaskFox />
```

Comment:
Are we sure about using the logos like this? @eriknson

Reply:
I think we can begin with them like this while we're in Flask, but we should for sure revisit this when moving to stable.